build.gradle.kts

```diff
@@ -24,27 +26,38 @@ subprojects {
     apply(plugin = "com.github.kt3k.coveralls")
 
     repositories {
-        jcenter()
+        mavenCentral()
     }
 
-    tasks.withType<JavaCompile> {
-        options.isDeprecation = true
-        options.isWarnings = true
-        if (JavaVersion.current().isJava9Compatible) {
-          options.compilerArgs = listOf("--release", "8", "-Xlint:unchecked")
-        }
-    }
-
-    configure<JavaPluginConvention> {
+    configure<JavaPluginExtension> {
         sourceCompatibility = JavaVersion.VERSION_1_8
         targetCompatibility = JavaVersion.VERSION_1_8
-        group = "org.roaringbitmap"
     }
 
-    tasks.named<JacocoReport>("jacocoTestReport") {
-        reports {
-            // used by coveralls
-            xml.isEnabled = true
+    group = "org.roaringbitmap"
+
+    tasks {
+        withType<JavaCompile> {
+            options.isDeprecation = true
+            options.isWarnings = true
+            if (JavaVersion.current().isJava9Compatible) {
+                options.compilerArgs = listOf("--release", "8", "-Xlint:unchecked")
+            }
+        }
+
+        named<JacocoReport>("jacocoTestReport") {
+            reports {
+                // used by coveralls
+                xml.isEnabled = true
+            }
+        }
+
+        withType<Javadoc> {
```

Comment:
drive-by squelching of the javadoc warning output since it makes other warnings harder to spot